features/google/src/views/report/report.jsx

```diff
@@ -2,11 +2,12 @@ import { RoutingWrapper } from "@polypoly-eu/poly-look";
 import React, { useContext, useState } from "react";
 import { useHistory } from "react-router-dom";
 import { GoogleContext } from "../../context/google-context.jsx";
+import i18n from "!silly-i18n";
 
 import "./report.css";
 
 const ReportView = ({ reportStories }) => {
-    const { setReportResult, handleBack } = useContext(GoogleContext);
+    const { handleBack, handleReportSent } = useContext(GoogleContext);
```

Comment:
I understand that you're using here `handle*` for symmetry, and I love symmetry. So no biggie with naming above. Still, just consider if just renaming this to setReportIsSent would be a big difference (and would save a function)

Reply:
Difference would be that in the next step, we need to save the bool in storage and this would happen in handleReportSent together with the state setting. Anticipating the next change basically :)

Reply:
So they're functionally equivalent *just for the time being*, correct?